### PR TITLE
[Hexagon] Add mobilenet test

### PR DESCRIPTION
**Pull request by mehrdadh**
This PR adds mobilenetv2-7 onnx model test running with graph executor on hexagon.

cc @kparzysz-quic @csullivan

### CODE REVIEW

File: python/tvm/relay/op/strategy/hexagon.py [hunk 1]
```diff
@@ -44,27 +44,49 @@ def conv2d_strategy_hexagon(attrs, inputs, out_type, target):
...
+            assert kernel_layout == "HWOI"
+            strategy.add_implementation(
+                wrap_compute_conv2d(topi.nn.depthwise_conv2d_nhwc),
+                wrap_topi_schedule(topi.hexagon.schedule_depthwise_conv2d_nhwc),
+                name="depthwise_conv2d_nhwc.generic",
+            )
+        else:
+            raise RuntimeError("Unsupported depthwise_conv2d layout {}".format(layout))
+    else:  # group_conv2d
+        raise RuntimeError("Unsupported group_conv2d layout {}".format(layout))
```
Comment (kparzysz-quic) [inline python/tvm/relay/op/strategy/hexagon.py:87]: You can use f-strings here.

File: python/tvm/relay/op/strategy/hexagon.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 from tvm import topi
 from .generic import *
 from .. import op as _op
-
+from .generic import is_depthwise_conv2d
```
Comment (kparzysz-quic) [inline python/tvm/relay/op/strategy/hexagon.py:24]: There is already an import of `*` from `.generic` at line 22.

**Review by kparzysz-quic — APPROVED** [review]
Couple of nitpicks.

File: python/tvm/relay/op/strategy/hexagon.py (hunk 1 above)
Comment (mehrdadh) [reply python/tvm/relay/op/strategy/hexagon.py]: done.

File: python/tvm/relay/op/strategy/hexagon.py (hunk 2 above)
Comment (mehrdadh) [reply python/tvm/relay/op/strategy/hexagon.py]: done.

**Review by csullivan — APPROVED** [review]
:shipit:

**Pull request merged**